<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IPBEC</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Directory"/>
    <addaction name="actionDark_File"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuRef_Images">
    <property name="title">
     <string>Ref Images</string>
    </property>
    <addaction name="actionClean"/>
    <addaction name="actionUse_Cleaned"/>
    <addaction name="actionUse_ROI_While_Cleaning"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRef_Images"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Directory">
   <property name="text">
    <string>Open Directory</string>
   </property>
   <property name="toolTip">
    <string>Change current directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDark_File">
   <property name="text">
    <string>Dark Image</string>
   </property>
   <property name="toolTip">
    <string>Select Dark Image</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Clean</string>
   </property>
  </action>
  <action name="actionUse_Cleaned">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Cleaned</string>
   </property>
  </action>
  <action name="actionUse_ROI_While_Cleaning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use ROI While Cleaning</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
